### ISSUE TITLE
Server error on several Artin rep pages

### ISSUE THREAD

**Issue by AndrewVSutherland** [LMFDB/lmfdb#4469]
All the Artin reps in this list of search results seem to be throwing server errors on both beta and production:

    https://www.lmfdb.org/ArtinRepresentation/?conductor=877534528&search_type=List

The problem is caused by trying to access properties of a WebNumberField object with its _data attribute set to None (I'm not sure how useful it is to create WebNumberField objects that are going to cause errors when one tries to use them in any nontrivial way, it might be better to catch the problem up front rather than down stream).

**Comment by jwj61** [comment]
As far as the test on _data being None, WebNumberField has to communicate if the given information did not produce a field in the database.  Whether you return None, throw an error, or return a WebNumberField object with information inside it seems to me to be six of one ... .  At one time, there was at least the idea that a WNF could do a few things even if the field was not in the database.  It could have produced its own dynamic knowl, although that is now done by a direct function.

I added a method to WNF is_in_db() which does the test for you.  It is in many ways equivalent, but better style since it keeps a barrier between the internal structure of the object and the outside world.  I used it here, but did not try to find other places to replace the test on _data since maybe people won't like this approach either.
